docs/conf.py

```diff
@@ -42,28 +38,28 @@
 html_theme = 'sphinx_rtd_theme'
 html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 html_static_path = ['_static']
-htmlhelp_basename = 'bigchaindb_driverdoc'
+htmlhelp_basename = 'bigchaindb_python_driverdoc'
 
 latex_elements = {}
 
 latex_documents = [
-    ('index', 'bigchaindb_driver.tex',
-     u'bigchaindb-driver Documentation',
-     u'BigchainDB', 'manual'),
+    ('index', 'bigchaindb_python_driver.tex',
+     'BigchainDB Python Driver Documentation',
+     'BigchainDB', 'manual'),
 ]
 
 man_pages = [
-    ('index', 'bigchaindb_driver',
-     u'bigchaindb-driver Documentation',
-     [u'BigchainDB'], 1)
+    ('index', 'bigchaindb_python_driver',
+     'BigchainDB Python Driver Documentation',
+     ['BigchainDB'], 1)
 ]
 
 texinfo_documents = [
-    ('index', 'bigchaindb_driver',
-     u'bigchaindb-driver Documentation',
-     u'BigchainDB',
-     'bigchaindb_driver',
-     'One line description of project.',
+    ('index', 'bigchaindb_python_driver',
+     'BigchainDB Python Driver Documentation',
+     'BigchainDB',
+     'bigchaindb_python_driver',
+     'A Python SDK for writing apps that communicate with BigchainDB',
```

Comment:
As an aside: I am not so sure about whether SDK is the appropriate term. Perhaps others (@r-marques @vrde @TimDaub @sohkai @diminator @rhsimplex) can share their thoughts.

We can however review this later.